config/dev.exs

```diff
@@ -39,7 +39,7 @@ config :apr, :basic_auth, [
 config :phoenix, :stacktrace_depth, 20
 
 config :kafka_ex,
-  brokers: [{"192.168.99.100", 9092}],
-  consumer_group: System.get_env("KAFKA_CONSUMER_GROUP") || "kafka_ex",
-  disable_default_worker: true,
+  brokers: [{"ip-10-0-0-49.ec2.internal", 9092}, {"ip-10-0-0-248.ec2.internal", 9092}],
```

Comment:
We actually had it originally as ENV variables, there was an issue when we were setting up CI where ENV variables had to be setup during compile time, so we switched back to this, but I guess with distillery we should test it again.